src/azure/cli/commands/parameters.py

```diff
@@ -65,6 +65,9 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
         return list(enum_type.__dict__[ENUM_MAPPING_KEY].keys())
     return completer
 
+def get_enum_choices(enum_type):
+    return [x.value for x in enum_type]
```

Comment:
I checked with Derek and he prefers to leave them separate.

---
In reply to: [74160422](https://github.com/Azure/azure-cli/pull/668#discussion_r74160422) [](ancestors = 74160422,74156155,74151895)